README.md

```diff
@@ -26,24 +26,24 @@ You can add this charts repo by:
 helm repo add skyscrapers https://skyscrapers.github.io/charts
 ```
 
-The actual Helm charts index is being hosted in GitHub pages in this same repo (`gh-pages` git branch). 
+The actual Helm charts index is being hosted in GitHub pages in this same repo (`gh-pages` git branch).
 The charts and the charts index are automatically updated by Concourse when new commits arrive
 on the master branch. see the [`ci`](https://github.com/skyscrapers/ci) repository for more details
 on the setup.
 
 ## Bootstrap base charts
 
-First you need to generate a `values.yaml` using the 
+First you need to generate a `values.yaml` using the
 [`terraform-kubernetes/base`](https://github.com/skyscrapers/terraform-kubernetes/tree/master/base)
 module, then you can just start installing:
 
 ```console
-helm upgrade --install kube2iam skyscrapers/kube2iam --values helm-values.yaml
-helm upgrade --install kube-lego skyscrapers/kube-lego --values helm-values.yaml
-helm upgrade --install nginx-ingress skyscrapers/nginx-ingress --values helm-values.yaml
-helm upgrade --install external-dns skyscrapers/external-dns --values helm-values.yaml
-helm upgrade --install kubesignin skyscrapers/kubesignin --values helm-values.yaml
+helm upgrade --install kube2iam kube2iam --namespace infrastructure --values helm-values.yaml
```

Comment:
Won't work yet. You need to pass a repo to install from. The stable Kubernetes repo is listed as `stable`:

```
$ helm repo list
NAME       	URL
stable     	https://kubernetes-charts.storage.googleapis.com
local      	http://127.0.0.1:8879/charts
skyscrapers	https://skyscrapers.github.io/charts
```
the chart names should read:
`stable/kube2iam` etc...